docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -1,351 +1,440 @@
+| **One layer upstream of an asset** | `1+key:"my_asset"` | Selects assets that directly provide input to `my_asset`. |
+| **Two layers upstream of an asset** | `2+key:"my_asset"` | Selects assets two steps upstream from `my_asset`. |
+| **All layers downstream of an asset** | `key:"my_asset"+` | Selects all downstream assets that depend on `my_asset`. |
+| **One layer downstream of an asset** | `key:"my_asset"+1` | Selects assets that directly receive input from `my_asset`. |
+| **Two layers downstream of an asset** | `key:"my_asset"+2` | Selects assets two steps downstream from `my_asset`. |
+| **One layer upstream and downstream of an asset** | `1+key:"my_asset"+1` | Selects one layer of assets providing input to and receiving input from `my_asset`. |
+| **Two layers upstream and downstream of an asset** | `2+key:"my_asset"+2` | Selects two layers of assets providing input to and receiving input from `my_asset`. |
+| **All layers upstream and downstream of an asset** | `+key:"my_asset"+` | Selects all assets upstream and downstream of `my_asset`. |
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/select-single-asset.png)
+### Operands and grouping
 
-</TabItem>
-</Tabs>
+You can combine multiple filters with operands and group them with parentheses to further refine your asset selection:
 
----
+| Operand | Syntax | Description |
+|---------|--------|-------------|
+| **`and`** | `owner:"alice" and kind:"dbt"` | Selects assets owned by `alice` of kind `dbt`. |
+| **`or`** | `owner:"billing" or owner:"sales"` | Selects assets owned by either `billing` or `sales`. |
+| **`not`** | `not tag:"obsolete"` | Excludes assets tagged with `obsolete`. |
+| **Grouping `()`** | `(owner:"alice" and kind:"table") or group:"analytics"` | Symbols `(` and `)` used to group expressions and control the order of evaluation in queries. This example selects assets that are both owned by `alice` and of kind `table`, or that belong to the `analytics` group. |
 
-### Selecting assets with multiple key components \{#multiple-key-components}
+### Functions
 
-To select an asset with a key containing multiple components, such as a prefix, insert slashes (`/`) between the components.
+You can use `sink()` and `root()` functions to return the sink and root assets of an asset selection.
 
-This example selects the `manhattan/manhattan_stats` asset, which is defined below:
+- **Sink assets** are assets without any downstream dependencies (leaf nodes), which means they don't provide input to any other assets.
+- **Root assets** are assets without any upstream dependencies (root nodes), which means no assets provide input to them.
 
-```python
-@asset(
-    deps=[AssetKey(["taxi_trips"]), AssetKey(["taxi_zones"])], key_prefix="manhattan"
-)
-def manhattan_stats(database: DuckDBResource):
- ...
-```
-
-<Tabs>
-<TabItem value="python" label="Python">
+| Function | Syntax | Description |
```

Comment:
Maybe `Function | Example | Description` 

roots | `roots(owner:"team:billing")` | Find all root assets depended on by the billing team
sinks | `sinks(owner:"team:billing")` | Find all sink assets depending on assets by the billing team